### PR TITLE
config,docker: switch to postgres-ha role

### PR DESCRIPTION
**Pull request by yakimant**
Needed for easier initialisation and better for maintenance. Also switched to compose v2.

### CODE REVIEW

File: templates/config/openkm.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # OpenKM Hibernate configuration values
 spring.datasource.driver-class-name=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://localhost:{{ openkm_db_cont_port | mandatory }}/{{ openkm_db_name | mandatory }}
+spring.datasource.url=jdbc:postgresql://localhost:{{ postgres_ha_cont_port | mandatory }}/{{ openkm_db_name | mandatory }}
```
Comment (jakubgs) [inline templates/config/openkm.properties]: Please don't do this. Roles should not mix variables between one another.

In order to pass this from PostgresHA role you need to provide it in the `group_vars` in the fleet repo.

File: defaults/main.yml [hunk 2]
```diff
@@ -34,11 +34,6 @@ openkm_xvfb_cont_vol: '{{ openkm_service_path }}/xvfb'
...
 openkm_xvfb_cont_uid: 0
 # DB
-openkm_db_cont_name: '{{ openkm_service_name }}-db'
-openkm_db_cont_vol: '{{ openkm_service_path }}/db'
-openkm_db_cont_image: 'postgres:16.1-alpine3.18'
-openkm_db_cont_port: 5432
-openkm_db_cont_uid: 70
 openkm_db_name: 'openkm'
 openkm_db_user: 'openkm'
 openkm_db_pass: 'changeIfYouCare'
```
Comment (jakubgs) [inline defaults/main.yml:41]:
```suggestion
openkm_db_addr: 'localhost'
openkm_db_post: 5432
openkm_db_name: 'openkm'
openkm_db_user: 'openkm'
openkm_db_pass: 'changeIfYouCare'
```

File: tasks/docker.yml [hunk 3]
```diff
@@ -1,20 +1,32 @@
...
-    build: true
-    state: '{{ compose_state }}'
-    recreate: '{{ compose_recreate | default("smart") }}'
-    restarted: >
-      {{ openkm_cont_env.changed or
-         openkm_config.changed or
-         compose_restart }}
+    files: '{{ openkm_compose.files | map(attribute="path") | list }}'
+    # TODO: get access to docker registry
+    #pull: 'always'
```
Comment (jakubgs) [inline tasks/docker.yml]: Probably should be removed right?

**Review by jakubgs — COMMENTED** [review]
Looks good, but you need to fix the integration with Postgres role, it should be done via `group_vars`.

File: tasks/docker.yml (hunk 3 above)
Comment (yakimant) [reply tasks/docker.yml]: Actually, we've got an access to the registry, i will update.

**Pull request merged**